main.go

```diff
@@ -1,9 +1,14 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/dachad/tcpgoon/cmd"
 )
 
+var version string
+
 func main() {
+	fmt.Println("Running tcpgoon version " + version)
```

Comment:
its fine if we are in the debug mode. If not, just expose the version if the user calls a -version (if that's the case, the program should only show the version and exit)